Dockerfile

```diff
@@ -1,5 +1,10 @@
-FROM rabbitmq:3.8.2-alpine
+FROM bitnami/rabbitmq:3.8
 
-RUN wget https://github.com/rabbitmq/rabbitmq-message-timestamp/releases/download/v3.8.0/rabbitmq_message_timestamp-3.8.0.ez && \
-    mv rabbitmq_message_timestamp-3.8.0.ez /plugins/
+USER root
 
+RUN apt-get update -y \
+    && apt-get install wget -y \
+    && wget https://github.com/rabbitmq/rabbitmq-message-timestamp/releases/download/v3.8.0/rabbitmq_message_timestamp-3.8.0.ez \
+    && mv rabbitmq_message_timestamp-3.8.0.ez /plugins
```

Comment:
Consider cleaning the apt cache to remove unnecessary content form the image.

Reply:
Consider adding `&& rm -rf /var/lib/apt/lists/*`

Reply:
done